### PR TITLE
Update Changelog.md

### PR DESCRIPTION
**Pull request by wthomson**
Spelling and syntax corrections

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
+- Added more flexibility with solder-on jumpers.
 
 ### Added [terminals add-on](https://github.com/srg74/WLED-wemos-shield/tree/master/resources/Add-ons)
 
 ### Added [linear audio input plug-in, and digital ICS-43434 microphone add-on boards](https://github.com/srg74/WLED-wemos-shield/tree/master/resources/Add-ons) for Shield v1.5 and up
 
 ### v1.51
 
-- Minor update. Fixed library and gerber files errors.
+- Minor update. Fixed library and gerber file errors.
```
Comment (srg74) [inline Changelog.md:26]: Gerber file is not single, it's a set of the files

File: Changelog.md [hunk 2]
```diff
@@ -32,8 +32,8 @@
 
 - Exposed additional pins for I2S microphones. :bangbang: ESP32 boards only :bangbang:
 - Removed header 3 positions labeled AUDIO.
-- Added header 6 poitions labeled SOUND.
-- Added second row of headers for ESP32 boards connectivities.
+- Added header 6 position labeled SOUND.
```
Comment (srg74) [inline Changelog.md:35]: Header have a 6 positions

Comment (wthomson) [reply Changelog.md:35]: True, they do indeed have six positions, but the text refers to a _single_ header, hence position as opposed to positions.

File: Changelog.md (hunk 1 above)
Comment (wthomson) [reply Changelog.md:26]: As with the header, the text refers to one set of files, hence file, as opposed to files.

**Review by wthomson — COMMENTED** [review]
I'm a native english speaker, so have folowed the grammar and syntax rules to the best of my ability. (I've been an Electronics Technician since 1974)

**Comment by srg74** [comment]
> I'm a native english speaker, so have folowed the grammar and syntax rules to the best of my ability. (I've been an Electronics Technician since 1974)

I highly appreciate your help and I think more like European do.

**Comment by wthomson** [comment]
Thank you very much for the kind words. If you like, I would be happy to review the rest of the documentation.

**Comment by srg74** [comment]
I have nothing against it. I appreciate any help with my project. This way project become more interesting and looks professional 😊

**Comment by wthomson** [comment]
Sounds good! thumbsup

BTW, my name is Bill. Nice to meet you.

**Pull request merged**